### ISSUE TITLE
ER_TRUNCATED_WRONG_VALUE: Truncated incorrect DOUBLE value: 'steam:11000010fddfbc0'

### ISSUE THREAD

**Issue by lowheartrate** [KASHZIN/kashacters#22]
I have installed the resource and made sure to following the installation instructions properly but when loading into the server I receive a blank loading screen like so:
![result](https://i.imgur.com/YSjnW1e.png)
I am receiving the following error in my server console when going to connect to the server as well:
![error](https://i.imgur.com/ydF1ZFC.png)

**Comment by lowheartrate** [comment]
I have made sure I have all of my `owner` and `identifier` column's being used in the **server/main.lua** as well:
```
local IdentifierTables = {
    {table = "users", column = "identifier"},
    {table = "user_accounts", column = "identifier"},
    {table = "user_contacts", column = "identifier"},
    {table = "user_inventory", column = "identifier"},
    {table = "user_parkings", column = "identifier"},
    {table = "society_moneywash", column = "identifier"},
    {table = "phone_users_contacts", column = "identifier"},
    {table = "jail", column = "identifier"},
    {table = "characters", column = "identifier"},
    {table = "billing", column = "identifier"},
    {table = "rented_vehicles", column = "owner"},
    {table = "addon_account_data", column = "owner"},
    {table = "addon_inventory_items", column = "owner"},
    {table = "datastore_data", column = "owner"}, 
    {table = "owned_vehicles", column = "owner"}, 
    {table = "phone_calls", column = "owner"}, 
    {table = "phone_messages", column = "owner"}, 
    {table = "rented_vehicles", column = "owner"}, 
    {table = "user_licenses", column = "owner"}, 
}
```

**Comment by lowheartrate** [comment]
Fixed this by using **mysql-async** version 3.0.1 instead of the latest release.

**Comment by lowheartrate** [comment]
Fixed this by using **mysql-async** version 3.0.1 instead of the latest release.
